### ISSUE TITLE
Failure running fpcupdeluxe: error executing sequence FPC, while trunk installation

### ISSUE THREAD

**Issue by billyeatcookies** [LongDirtyAnimAlf/fpcupdeluxe#363]
I got an error while trying to install trunk version of both lazarus and fpc 
![image](https://user-images.githubusercontent.com/70792552/110276599-61c01900-7fcb-11eb-9e87-876ccc4d13a1.png)

![image](https://user-images.githubusercontent.com/70792552/110276539-3a694c00-7fcb-11eb-9a5f-d441b4b63902.png)

```
FPCUP(deluxe) is starting up.

FPCupdeluxe basedir:       C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe
Binutils dir:              C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap
Bootstrap dir:             C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap

FPC URL:                   https://svn.freepascal.org/svn/fpc/trunk
FPC source directory:      C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc
FPC install directory:     C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpc
FPC options:               

Lazarus URL:               https://svn.freepascal.org/svn/lazarus/trunk
Lazarus source directory:  C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus
Lazarus install directory: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus
Lazarus options:           -g -gl -O1

Please stand back and enjoy !

3/8/2021 10:05:32: fpcupdeluxe: V322 (20210301) started.
FPCUPdeluxe V1.8.2m for x86_64-win64 running on Win64-10.0.19042

Checking dev-libs for: win32
fpcupdeluxe: info: FPCNativeInstaller (CleanModule: FPC): No FPC source directory (C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc) found [yet] ... nothing to be done
fpcupdeluxe: info: FPCNativeInstaller (CheckAndGetTools): Getting OpenSLL library files.
fpcupdeluxe: WARNING: FPCNativeInstaller (DownloadOpenSSL): No OpenSLL library files available for SSL. Going to download them.
fpcupdeluxe: info: FPCNativeInstaller (GetFile): Downloading https://svn.freepascal.org/svn/lazarus/binaries/x86_64-win64/openssl/libeay32.dll
Download progress libeay32.dll: Starting download.
Download progress libeay32.dll: 1.4 MB
Download progress libeay32.dll: download ready !
fpcupdeluxe: info: FPCNativeInstaller (GetFile): Downloading https://svn.freepascal.org/svn/lazarus/binaries/x86_64-win64/openssl/ssleay32.dll
Download progress ssleay32.dll: Starting download.
Download progress ssleay32.dll: download ready !
fpcupdeluxe: WARNING: FPCNativeInstaller (GetFile): Direct OpenSLL library files download from https://svn.freepascal.org/svn/lazarus/binaries/x86_64-win64/openssl ok.
fpcupdeluxe: info: FPCNativeInstaller (FindSVNSubDirs): Could not find svn executable in or under C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn
fpcupdeluxe: info: FPCNativeInstaller (FindSVNSubDirs): Going to download SVN
fpcupdeluxe: info: FPCNativeInstaller (GetFile): Downloading https://www.visualsvn.com/files/Apache-Subversion-1.12.2.zip
Download progress FPCUPTMP00000.zip: Starting download.
Download progress FPCUPTMP00000.zip: 1.3 MB
Download progress FPCUPTMP00000.zip: 2.5 MB
Download progress FPCUPTMP00000.zip: 3.6 MB
Download progress FPCUPTMP00000.zip: download ready !
Extracted #1. File svn.exe out of #31
Extracted #2. File svnadmin.exe out of #31
Extracted #3. File svnbench.exe out of #31
Extracted #4. File svndumpfilter.exe out of #31
Extracted #5. File svnfsfs.exe out of #31
Extracted #6. File svnlook.exe out of #31
Extracted #7. File svnserve.exe out of #31
Extracted #8. File svnsync.exe out of #31
Extracted #9. File svnversion.exe out of #31
Extracted #10. File svnmucc.exe out of #31
Extracted #11. File svnrdump.exe out of #31
Extracted #12. File libsvn_client-1.dll out of #31
Extracted #13. File libsvn_delta-1.dll out of #31
Extracted #14. File libsvn_diff-1.dll out of #31
Extracted #15. File libsvn_fs-1.dll out of #31
Extracted #16. File libsvn_fs_fs-1.dll out of #31
Extracted #17. File libsvn_fs_x-1.dll out of #31
Extracted #18. File libsvn_fs_util-1.dll out of #31
Extracted #19. File libsvn_ra-1.dll out of #31
Extracted #20. File libsvn_repos-1.dll out of #31
Extracted #21. File libsvn_subr-1.dll out of #31
Extracted #22. File libsvn_wc-1.dll out of #31
Extracted #23. File libapr-1.dll out of #31
Extracted #24. File libaprutil-1.dll out of #31
Extracted #25. File msvcp100.dll out of #31
Extracted #26. File msvcr100.dll out of #31
Extracted #27. File APR License.txt out of #31
Extracted #28. File APR Util License.txt out of #31
Extracted #29. File OpenSSL License.txt out of #31
Extracted #30. File Subversion License.txt out of #31
Extracted #31. File ZLib License.txt out of #31
Extracted #all. Ready extracting.
fpcupdeluxe: info: FPCNativeInstaller (GetFile): SVN download and unpacking ok. Not going to search SVN client itself in C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn
Found valid svn.exe application.
fpcupdeluxe: info: FPCNativeInstaller (FindSVNSubDirs): Getting Wget.
fpcupdeluxe: info: FPCNativeInstaller (DownloadWget): No Wget found. Going to download it.
fpcupdeluxe: info: FPCNativeInstaller (GetFile): Downloading https://eternallybored.org/misc/wget/1.20.3/64/wget.exe
Download progress wget.exe: Starting download.
Download progress wget.exe: 740 KB
Download progress wget.exe: 756 KB
Download progress wget.exe: 1.4 MB
Download progress wget.exe: 1.9 MB
Download progress wget.exe: 1.9 MB
Download progress wget.exe: 2.3 MB
Download progress wget.exe: 3.6 MB
Download progress wget.exe: download ready !
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe export --quiet --force --non-interactive --trust-server-cert -r HEAD https://svn.freepascal.org/svn/fpcbuild/tags/release_3_2_0/install/binw32/patch.exe/ C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap (working dir: )
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe export --quiet --force --non-interactive --trust-server-cert -r HEAD https://svn.freepascal.org/svn/fpcbuild/tags/release_3_2_0/install/binw32/patch.exe.manifest/ C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
fpcupdeluxe: info: FPCNativeInstaller (GetFile): Downloading https://downloads.sourceforge.net/project/sevenzip/7-Zip/9.20/7za920.zip
Using PowerShell to download C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\7Zip\7za920.zip
Using WinINet to download C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\7Zip\7za920.zip
Extracted #1. File 7za.exe out of #1
Extracted #all. Ready extracting.
Extracted #all. Ready extracting.
fpcupdeluxe: info: FPCNativeInstaller (GetFile): Downloading https://downloads.sourceforge.net/project/gnuwin32/unrar/3.4.3/unrar-3.4.3-bin.zip
Using PowerShell to download C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\unrar\unrar-3.4.3-bin.zip
Extracted #5 files out of #69
Extracted #10 files out of #69
Extracted #15 files out of #69
Extracted #20 files out of #69
Extracted #25 files out of #69
Extracted #30 files out of #69
Extracted #35 files out of #69
Extracted #40 files out of #69
Extracted #45 files out of #69
Extracted #50 files out of #69
Extracted #55 files out of #69
Extracted #60 files out of #69
Extracted #65 files out of #69
Found valid git.exe application.
fpcupdeluxe: info: FPCNativeInstaller (GetFile): HG (mercurial) client not found. Downloading it (may take time) from https://github.com/LongDirtyAnimAlf/fpcupdeluxe/releases/download/HG-4.7/hg64.zip
fpcupdeluxe: info: FPCNativeInstaller (GetFile): Downloading https://github.com/LongDirtyAnimAlf/fpcupdeluxe/releases/download/HG-4.7/hg64.zip
Download progress hg64.zip: Starting download.
Download progress hg64.zip: 1.5 MB
Download progress hg64.zip: 1.6 MB
Download progress hg64.zip: 2.4 MB
Download progress hg64.zip: 3.3 MB
Download progress hg64.zip: 3.4 MB
Download progress hg64.zip: 3.4 MB
Download progress hg64.zip: 4.5 MB
Download progress hg64.zip: 4.5 MB
Download progress hg64.zip: 5.8 MB
Download progress hg64.zip: 6.2 MB
Download progress hg64.zip: 7.1 MB
Download progress hg64.zip: 8.4 MB
Download progress hg64.zip: 8.8 MB
Download progress hg64.zip: 9.3 MB
Download progress hg64.zip: download ready !
fpcupdeluxe: info: FPCNativeInstaller (GetFile): HG download ready: unpacking (may take time).
Extracted #5 files out of #54
Extracted #10 files out of #54
Extracted #15 files out of #54
Extracted #20 files out of #54
Extracted #25 files out of #54
Extracted #30 files out of #54
Extracted #35 files out of #54
Extracted #40 files out of #54
Extracted #45 files out of #54
Extracted #50 files out of #54
Extracted #all. Ready extracting.
Found valid hg.exe application.
fpcupdeluxe: info: FPCNativeInstaller (GetModule: FPC): Start checkout/update of FPC sources.
fpcupdeluxe: info: Directory C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc is not an SVN repository (or a repository with the wrong remote URL).
fpcupdeluxe: info: FPCNativeInstaller (DownloadFromSVN: FPC): Creating directory C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc for SVN checkout.
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
fpcupdeluxe: info: FPCNativeInstaller (DownloadFromSVN: FPC): Running SVN checkout or update.
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E155007: 'C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc' is not a working copy
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E155007: 'C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc' is not a working copy
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe checkout --quiet --non-interactive --trust-server-cert -r HEAD https://svn.freepascal.org/svn/fpc/trunk/ C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E120106: ra_serf: The server sent a truncated HTTP response body.
fpcupdeluxe: WARNING: SVN client error return code: 1
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe checkout --quiet --non-interactive --trust-server-cert -r HEAD https://svn.freepascal.org/svn/fpc/trunk/ C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E155004: Run 'svn cleanup' to remove locks (type 'svn help cleanup' for details)
svn: E155004: Working copy 'C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc' locked.
svn: E155004: 'C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc' is already locked.
fpcupdeluxe: WARNING: SVN client error return code: 1
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe cleanup --non-interactive C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe cleanup --non-interactive C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe update --quiet --non-interactive --trust-server-cert -r HEAD C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E120108: Error running context: The server unexpectedly closed the connection.
fpcupdeluxe: ERROR: SVN client error return code: 1
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe checkout --quiet --non-interactive --trust-server-cert -r HEAD https://svn.freepascal.org/svn/fpc/trunk/ C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E155004: Run 'svn cleanup' to remove locks (type 'svn help cleanup' for details)
svn: E155004: Working copy 'C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc' locked.
svn: E155004: 'C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc' is already locked.
fpcupdeluxe: WARNING: SVN client error return code: 1
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe cleanup --non-interactive C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe cleanup --non-interactive C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe update --quiet --non-interactive --trust-server-cert -r HEAD C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E120108: Error running context: The server unexpectedly closed the connection.
fpcupdeluxe: ERROR: SVN client error return code: 1
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe checkout --quiet --non-interactive --trust-server-cert -r HEAD https://svn.freepascal.org/svn/fpc/trunk/ C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E155004: Run 'svn cleanup' to remove locks (type 'svn help cleanup' for details)
svn: E155004: Working copy 'C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc' locked.
svn: E155004: 'C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc' is already locked.
fpcupdeluxe: WARNING: SVN client error return code: 1
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe cleanup --non-interactive C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe update --quiet --non-interactive --trust-server-cert -r HEAD C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
svn: E730054: Error running context: An existing connection was forcibly closed by the remote host.  
fpcupdeluxe: ERROR: SVN client error return code: 1
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc (working dir: C:\Users\billy\Downloads\fpcupdeluxx\)
fpcupdeluxe: info: FPCNativeInstaller (DownloadFromSVN: FPC): SVN gave error code: 1

fpcupdeluxe: info: FPCNativeInstaller (DownloadFromSVN: FPC): SVN gave error message: Path: fpcupdeluxe\fpcsrc
Working Copy Root Path: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc
URL: https://svn.freepascal.org/svn/fpc/trunk
Relative URL: ^/trunk
Repository Root: https://svn.freepascal.org/svn/fpc
Repository UUID: 3ad0048d-3df7-0310-abae-a5850022a9f2
Revision: 48906
Node Kind: directory
Schedule: normal

fpcupdeluxe: ERROR: FPCNativeInstaller (GetModule: FPC): Checkout/update of FPC sources failure.


ERROR: Fpcupdeluxe fatal error !
Sequencer (FPC): Failure running fpcupdeluxe: error executing sequence FPC
Sequencer (Default): Failure running fpcupdeluxe: error executing sequence Default
```

**Comment by LongDirtyAnimAlf** [comment]
This looks to be a more or less simple svn error.
Please remove the fpcsrc directory and run again.

**Comment by billyeatcookies** [comment]
the fpc issue is fixed, but another error is occurring while lazarus installation.

```
Going to install/update Lazarus only with given options.
FPCUP(deluxe) is starting up.

FPCupdeluxe basedir:       C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe
Binutils dir:              C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap
Bootstrap dir:             C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap

FPC URL:                   https://svn.freepascal.org/svn/fpc/trunk
FPC source directory:      C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcsrc
FPC install directory:     C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpc
FPC options:               

Lazarus URL:               https://svn.freepascal.org/svn/lazarus/trunk
Lazarus source directory:  C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus
Lazarus install directory: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus
Lazarus options:           -g -gl -O1

Please stand back and enjoy !

3/8/2021 17:07:45: fpcupdeluxe: V322 (20210301) started.
FPCUPdeluxe V1.8.2m for x86_64-win64 running on Win64-10.0.19042

Checking dev-libs for: win32
Found valid svn.exe application.
Found valid git.exe application.
Found valid hg.exe application.
Found valid make.exe application.
fpcupdeluxe: info: LazarusNativeInstaller (CleanModule: Lazarus): Deleting Lazarus primary config file (lazarus.cfg).
fpcupdeluxe: info: LazarusNativeInstaller (CleanModule: Lazarus): Deleting Lazarus primary config file (lazarus.cfg).
fpcupdeluxe: info: LazarusNativeInstaller (CleanModule: Lazarus): Running "make distclean" twice inside C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\make.exe FPC=C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpc\bin\x86_64-win64\fpc.exe PP=C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpc\bin\x86_64-win64\ppcx64.exe PREFIX=C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus INSTALL_PREFIX=C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus LAZARUS_INSTALL_DIR=C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus\ UPXPROG=echo OS_SOURCE=win64 CPU_SOURCE=x86_64 OS_TARGET=win64 CPU_TARGET=x86_64 --directory=C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus distclean (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\make.exe: *** No rule to make target `distclean'.  Stop.
fpcupdeluxe: info: LazarusNativeInstaller (CheckModule: Lazarus): Checking Lazarus sources with TSVNClient
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
fpcupdeluxe: info: LazarusNativeInstaller (CheckModule: Lazarus): sources ok.
fpcupdeluxe: info: LazarusNativeInstaller (GetModule: Lazarus): Start checkout/update of Lazarus sources.
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe status --depth infinity C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
fpcupdeluxe: info: LazarusNativeInstaller (DownloadFromSVN: Lazarus): Running SVN checkout or update.
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe cleanup --non-interactive C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe update --quiet --non-interactive --trust-server-cert -r HEAD C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
fpcupdeluxe: ERROR: SVN client error return code: 1
fpcupdeluxe: Executing: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\fpcbootstrap\svn\bin\svn.exe info C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus (working dir: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus)
fpcupdeluxe: info: LazarusNativeInstaller (DownloadFromSVN: Lazarus): SVN gave error code: 1

fpcupdeluxe: info: LazarusNativeInstaller (DownloadFromSVN: Lazarus): SVN gave error message: Path: .
Working Copy Root Path: C:\Users\billy\Downloads\fpcupdeluxx\fpcupdeluxe\lazarus
URL: https://svn.freepascal.org/svn/lazarus/trunk
Relative URL: ^/trunk
Repository Root: https://svn.freepascal.org/svn/lazarus
Repository UUID: 4005530d-fff6-0310-9dd1-cebe43e6787f
Revision: 64768
Node Kind: directory
Schedule: normal

fpcupdeluxe: ERROR: LazarusNativeInstaller (GetModule: Lazarus): Checkout/update of Lazarus sources failure.


ERROR: Fpcupdeluxe fatal error !
Sequencer (Lazarus): Failure running fpcupdeluxe: error executing sequence Lazarus
Sequencer (Only): Failure running fpcupdeluxe: error executing sequence Only
```

**Comment by LongDirtyAnimAlf** [comment]
As this is related to svn, I am closing due to the fact that svn is no longer used by FPC or Lazarus.

**Comment by fdridder** [comment]
I got the same error with the latest version of fcpupdeluxe, v2.4.0f, on MacOS.

I managed to reproduce it, it occurs when you recompile Lazarus in an already existing install path containing files of a previous build.
In this case it can be solved by removing the directory "lazarus"  from the install path.

It also occurs when you activate under Setup+ the option "Split Lazarus source and bins".
I tested it multiple times, and this option does not seem to work at all on MacOS.
